src/dlgIRC.cpp

```diff
@@ -675,7 +681,9 @@ void dlgIRC::slot_partedChannel(IrcPartMessage* message)
 
     if (message->isOwn()) {
         const QString luaText = IrcMessageFormatter::formatMessage(static_cast<IrcMessage*>(message), true);
-        mpHost->postIrcMessage(message->nick(), message->channel(), luaText);
+        if (mpHost) {
```

Comment:
Like last comment - why not just bail out at the start of the method if `mpHost` is a `nullptr`?